README.md

```diff
@@ -57,11 +54,9 @@ The following tasks are available:
 | Command | Description | Examples |
 | --- | --- | --- |
 | `clean` | Cleans all output files in the dist directory | `yarn clean` |
-| `build` | Cleans the dist directory, copies website assets and then brings in the rsp, express, and workflow icons into dist | `yarn build` |
-| `build:icons` | Deletes and rebuilds the icon folder, can only be run by an Adobe employee with access | `yarn build:icons` |
-| `build:site` | Cleans the dist directory, copies website assets and prepares the dist output for the site preview | `yarn build:site` |
+| `build` | Deletes and rebuilds the icon folder, can only be run by an Adobe employee with access | `yarn build:icons` |
```

Comment:
@castastrophe Little typo, I guess. The command at the end of the line is just `yarn build`.

Reply:
Great catch thank you!